env_manager/backends/venv_interface.py

```diff
@@ -138,6 +138,21 @@ def uninstall_packages(
         except subprocess.CalledProcessError as error:
             return (False, f"{error.returncode}: {error.stderr}")
 
+    def update_packages(
+        self, environment_path, packages, force=False, capture_output=False
+    ):
+        if os.name == "nt":
+            executable_path = Path(environment_path) / "Scripts" / "python.exe"
+        else:
+            executable_path = Path(environment_path) / "bin" / "python"
```

Comment:
I think you could declare `executable_path` as a `@property` method of this class, and use in all methods where it's defined in exactly the same way. That would make the code much simpler.